root/base/release.tx

```diff
@@ -9,7 +9,7 @@
 %%  override breadcrumbs -> {
 <div class="breadcrumbs">
   <span>
-    <a data-keyboard-shortcut="g a" rel="author" href="/author/[% $release.author %]" title="[% $author.asciiname %]" class="author-name">[% $author.name %]</a>
+    <a data-keyboard-shortcut="g a" rel="author" href="/author/[% $release.author %]" class="author-name">[% $author.display_name %]</a>
```

Comment:
Any reason for removing the title here? Is that useful for accessibility?

Reply:
Generally, the two names will be the same. And I'm not sure that a purely ASCII name is of any value for our use case.